parsing/src/processor.ts

```diff
@@ -569,6 +629,8 @@ async function processFile(inputFile: string, outputFile: string) {
         // Write the output to the file
         await fs.writeJson(path.resolve(outputFile), finalData, {spaces: 4});
         console.log(`Processed parts, buildings, and recipes have been written to ${outputFile}.`);
+
+        return items.parts;
```

Comment:
I'd just return `finalData` then in the tests you can access everything

Reply:
Yep! That is next. Was just trying to get the first test to pass! Now I'm going nuts. 🥜

Reply:
I identify as an ant eater and I like to eat those bugs! 😂